lib/krikri/logger.rb

```diff
@@ -0,0 +1,38 @@
+module Krikri
+  ##
+  # An application-wide tagged logger
+  # 
+  # @example with default logger
+  #   Krikri::Logger.log :info, 'message'
+  #
+  # @example with custom logger
+  #   logger = Logger.new(my_logger)
+  #   logger.log :info, 'message'
+  class Logger
+    ##
+    # @param [ActiveSupport::TaggedLogging] logger
+    def initialize(logger = ActiveSupport::TaggedLogging.new(Rails.logger))
```

Comment:
I somehow managed to overlook the YARD annotations here when I first reviewed it.

![](https://media.giphy.com/media/3Mm6HZud4m2oU/giphy.gif)